ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -305,63 +305,70 @@ export const RenewalCardBody = (props: {
     ],
     [
       AccessModule.PUBLICATIONCONFIRMATION,
-      () => (
```

Comment:
this is almost all whitespace/indents

Reply:
Thanks for that. Whitespace-ignoring diff helped.